pages/sources.md

```diff
@@ -1,35 +1,28 @@
-# Sources
-
-This is where your source analysis of any/all data should go. Delete this introductory material when submitting your assignment.
-
-This source analysis should discuss whatever source(s) you used for this project.
+{% include nav.html %} {% include css.html %}
 
-See the [README.md](../README.md) file for information on how to submit this assignment.
+# Sources
 
-**You SHOULD revise your source analysis if you already did one for the given website.**
----
 
 ## Basic Source Information
 
-* Where is your source located (URL? Or traditional media)
-* What kind of source is it? (Diaries, records of objects, data tables, etc...)
-* Who, or what organization, is responsible for hosting the source (if online), who gathered the information?
+* [https://www.slavevoyages.org/](https://www.slavevoyages.org/)
+* Database on the embarkation and debarkation of Trade routes in the Atlantic
+* National Endowment for the Humanities
 
 ---
 
 ## Authorship
 
-* Is there any information available discussing the process by which the information was gathered, or how the digital project was constructed?
-* How transparent is the project about the choices they made in gathering and transforming the data for digitalization? [Click here for an example of a an excellent 'About' page from a database about Anglo-Saxons](http://www.pase.ac.uk/about/index.html)
-* Do the organizations or individuals involved in gathering and publishing your data set have any stated goals or agendas?
-* Do the organizations or individuals involved seem to have any unstated agendas or goals?
-* If a 3rd party funded the project, who are they? Do they have any particular perspective?
+* The information gathered in the database in the combination of several years of independent and group collaboration mostly pulling resources from libraries and archives.
```

Comment:
Given the copious amount of information on the original website, you can say a lot more than this. Do they talk about any of their decisions in the data set? Or where the records come from, or any judgement calls made?